README.md

```diff
@@ -731,6 +731,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [FitriRibbit](https://github.com/FitriRibbit)
   - [Fiyinfoluwa Fatuase](https://github.com/Fiiyinfoluwa)
   - [Fitri Ribbit](https://github.com/FitriRibbit)
+  - [Flynn Salbenblatt](https://github.com/Skagra42)
```

Comment:
Please add your name in line 733, after Fiyinfoluwa Fatuase.
Let us know if you need any help with that. 🙂